### PR TITLE
Bump pyinstaller to 3.6 - CVE-2019-16784

### PR DESCRIPTION
**Pull request by LefterisJP**
## Description

Upgrade is needed to fix a vulnerability:
https://github.com/advisories/GHSA-7fcj-pq9j-wh2r

It's Windows only so it does not affect Raiden, but upgrading should be good nonetheless.

@ulope take a look please

## PR review check list

Quality check list that cannot be automatically verified.

- Safety
    - [ ] The changes respect the necessary conditions for safety (https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#protocol-values-constraints)
-  Code quality
    - [ ] Error conditions are handled
    - [ ] Exceptions are propagated to the correct parent greenlet
    - [ ] Exceptions are correctly classified as recoverable or unrecoverable
- Compatibility
    - [ ] State changes are forward compatible
    - [ ] Transport messages are backwards and forward compatible
- Commits
    - [ ] Have good messages
    - [ ] Squashed unecessary commits
- If it's a bug fix:
    - Regression test for the bug
        - [ ] Properly covers the bug
        - [ ] If an integration test is used, it could not be written as a unit test
- Documentation
    - [ ] A new CLI flag was introduced, is there documentation that explains usage?
- Specs
    - [ ] If this is a protocol change, are the specs updated accordingly? If so, please link PR from the specs repo.
- Is it a user facing feature/bug fix?
    - [ ] Changelog entry has been added

### CODE REVIEW

File: requirements/requirements-ci.txt [hunk 1]
```diff
@@ -121,7 +121,7 @@ pycryptodome==3.8.2
 pyflakes==2.1.1
 pygments==2.4.2
 pyhamcrest==1.9.0
-pyinstaller==3.4
+pyinstaller==3.6
```
Comment (palango) [inline requirements/requirements-ci.txt]: This pinning should go into `requirements-ci.in` and then run `requirements/deps upgrade pyinstaller`

Comment (LefterisJP) [reply requirements/requirements-ci.txt]: Ah damn. Thanks. To be fair I knew there was something like that after reading the readme but I tried `requirements/deps upgrade pyinstaller==3.6 ` and got a failure. Let me do that then.

Comment (LefterisJP) [reply requirements/requirements-ci.txt]: done

File: requirements/requirements-ci.in [hunk 2]
```diff
@@ -1,4 +1,4 @@
 -r requirements-dev.txt
 
-pyinstaller
+pyinstaller==3.6
 s3cmd
```
[the remark this answers is not in the document]
Comment (LefterisJP) [reply requirements/requirements-ci.in]: Understood. Let me fix that. I need to read about this way of managing python dependencies as I don't think I properly understand it yet.

**Pull request merged**